clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchUtil.scala

```diff
@@ -56,5 +56,18 @@ object ElasticsearchUtil {
       */
     def unsafeGet[T: Decoder](key: String): T =
       json.hcursor.get[T](key).fold(throw _, identity)
+
+    def getAsString(key: String): String = {
+      json.asObject
+        .flatMap(_.apply(key))
+        .map {
+          case s if s.isString =>
+            s.asString.getOrElse("")
+          case a if a.isArray =>
+            a.asArray.map(_.mkString(".")).getOrElse("")
+          case j => j.toString()
+        }
+        .getOrElse("")
```

Comment:
`getOrElse("")` worries me, it's too easy to silently corrupt data. I'd rather we explode if we get an unexpected value.